### PR TITLE
Whatsnew update for week of May 11

### PR DESCRIPTION
**Pull request by jfrontain**
## Purpose of this pull request

This pull request (PR) updates the whatsnew info feed for the week of may 11 - 14.

### CODE REVIEW

**Comment by jfrontain** [comment]
running tests

**Comment by jfrontain** [comment]
running tests

File: src/_data/whats-new.yml [hunk 1]
```diff
@@ -1,11 +1,48 @@
 title: Whats New on Devdocs
-description: |
+description:
```
Comment (dshevtsov) [inline src/_data/whats-new.yml:2]: There was a reason why this pipeline was there. Whatsnew updates can update only `updated` and `etnries`.

File: Rakefile [hunk 2]
```diff
@@ -91,4 +91,4 @@ task :convert do
   result = `bin/kramdown --input=html --output=kramdown`
   puts 'Converted text:'.magenta
   puts result.bold
-end
+end
```
Comment (dshevtsov) [inline Rakefile:94]: Whatsnew updated PR must not change any other files but `src/_data/whats-new.yml`, especially when this is an infrastructure file. I don't believe that an author and a reviewer didn't notice this change.

File: src/_data/whats-new.yml [hunk 3]
```diff
@@ -1,11 +1,48 @@
...
+  type: Major Update
+  date: May 12, 2020
+  link: https://github.com/magento/devdocs/pull/7195
+- description: Updated the _Cloud Guide_ to reflect the availability of Microsoft
+    Azure as an Iaas provider for Magento Commerce Cloud Pro projects.
+  versions: 2.x
+  type: Technical
+  date: May 12, 2020
+  link: https://github.com/magento/devdocs/pull/7199
+- description: Added Release Notes and updated Merchant Documentation for the Amazon
```
Comment (dshevtsov) [inline src/_data/whats-new.yml:27]: This should contain links to the updated topics.

File: src/_data/whats-new.yml [hunk 4]
```diff
@@ -1,11 +1,48 @@
...
+    Sales Channel 4.1.0 release.
+  versions: ''
```
Comment (dshevtsov) [inline src/_data/whats-new.yml:29]: Version should be specified.

File: src/_data/whats-new.yml [hunk 5]
```diff
@@ -1,11 +1,48 @@
...
+  type: Major Update
+  date: May 11, 2020
+  link: https://github.com/magento/devdocs/pull/7193
+- description: Added [Bypass Fastly](https://devdocs.magento.com/cloud/cdn/fastly-vcl-bypass-to-origin.html)
+    topic with instructions for creating a custom VCL snippet to bypass Fastly on
+    requests from a specific IP address, URL, or URL pattern.
+  versions: ''
```
Comment (dshevtsov) [inline src/_data/whats-new.yml:36]: Version should be specified.

**Pull request merged**